### ISSUE TITLE
false alarm by RuleAssertionMessage

### ISSUE THREAD

**Issue by yegor256** [volodya-lombrozo/jtcop#413]
This is my code:

```
    @Test
    void runsInDirectory() throws Exception {
        try (Mktemp temp = new Mktemp()) {
            final Path file = temp.path().resolve("test.txt");
            Files.write(file, "Hello, world!".getBytes());
            MatcherAssert.assertThat(
                "file must be written",
                file.toFile().exists(),
                Matchers.is(true)
            );
        }
    }
```

I'm getting this:

```
[ERROR] 	1) Method runsInDirectory doesn't have assertion statements.
[ERROR] 	Please add at least one assertion statement to the test method.
[ERROR] 	You can also ignore the rule by adding @SuppressWarnings("JTCOP.RuleAssertionMessage") annotation.
[ERROR] 	Rule: RuleAssertionMessage.
[ERROR] 	You can read more about the rule here: https://github.com/volodya-lombrozo/jtcop/blob/main/docs/rules/no-assertions.md
```

It looks like a false positive, because the test method does have an assertion.

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you try it one more time with `1.3.1` version please?
